pkg/apps/apps.go

```diff
@@ -43,7 +43,7 @@ const (
 const (
 	OnboardingSlug = "onboarding"
 	HomeSlug       = "home"
-	FilesSlug       = "files"
+	FilesSlug      = "files"
```

Comment:
So sorry about this one. :(

(Great example of what kind of typos aligned values may cause, I am not a big fan but I swear it was an accident)